utils/src/types/byte_string.rs

```diff
@@ -0,0 +1,116 @@
+use crate::buffer::{Buffer, Writable};
+use crate::decode::*;
+
+use super::I32;
+
+#[cfg_attr(test, derive(Debug, PartialEq))]
+pub struct ByteString {
+    pub length: I32,
```

Comment:
Q: Do we want to have the value of the `ByteString` here?

Reply:
Due to the limitations of stack memory size, and we currently do not display `ByteString`, we are not saving it. But if in the future we want to display more infos, perhaps we can use NVM to store transaction data.